### PR TITLE
Make glossary entries consistent

### PR DESCRIPTION
**Pull request by Prince213**
Unifies the style.

### CODE REVIEW

File: doc/Language/glossary.pod6 [hunk 1]
```diff
@@ -4,8 +4,8 @@
 
 =SUBTITLE Glossary of Raku terminology
 
-X<|Abstract Class>
 =head1 Abstract class
```
Comment (JJ) [inline doc/Language/glossary.pod6:7]: Sorry, been there, done that. Anchors must always be on top of the word, or the word itself it defines will be hidden. They should stay on top.

Comment (Prince213) [reply doc/Language/glossary.pod6]: Done.

File: doc/Language/glossary.pod6 [hunk 2]
```diff
@@ -167,8 +167,8 @@ sub the-answer() { 42 }                            # arity of 0
 The arity of a C<Callable> is one of the main selectors in
 L<multi-dispatch|#Multi-dispatch>.
 
-=head1 ASCII operator
 X<|ASCII operator>
+=head1 ASCII operator
 X<|Texas operator>
 =begin comment
```
Comment (JJ) [inline doc/Language/glossary.pod6:173]: It's probably sensible to just eliminate that index, together with the comment.

Comment (Prince213) [reply doc/Language/glossary.pod6:173]: Do you mean `Texas operator`?

**Comment by JJ** [comment]
Yep. Anyway, it's merged, it was rather a comment for myself.

**Pull request merged**